olive/passes/onnx/optimum_merging.py

```diff
@@ -21,6 +21,13 @@ class OptimumMerging(Pass):
 
     _accepts_composite_model = True
 
+    @staticmethod
+    def is_accelerator_agnostic(accelerator_spec: AcceleratorSpec) -> bool:
```

Comment:
is this pass accelerator agnostic? I only see the EP when used to generate the model?

Reply:
![image](https://github.com/microsoft/Olive/assets/13343117/f5a4860b-1b0d-4108-8a56-51637edc4ce9)
For this case, EP is used to optimize model graph. I suspect that there might be different optimizations wrt EPs.